### ISSUE TITLE
Confusing error message when using `build-custom`

### ISSUE THREAD

**Issue by jfy133** [pirovc/ganon#244]
Hello,

I'm testing a conda-installed `ganon` 1.5 on [this](https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/sarscov2/genome/genome.fasta) FASTA File.

However when running with the following command I get the error:

```bash
$ ganon build-custom -i genome.fasta -d out/ --verbose
- - - - - - - - - -
   _  _  _  _  _   
  (_|(_|| |(_)| |  
   _|   v. 1.5.0
- - - - - - - - - -
Total valid files: 1

Downloading and parsing ncbi taxonomy
 - done in 20.09s.

Parsing sequences from --input (1 files)
 - 1 unique entries
 - done in 0.01s.

Retrieving sequence information from NCBI e-utils
The following command failed to run:
 -i out/_files/build/accessions.txt -k -e 
[Errno 2] No such file or directory: '-i'
Error code: 1
```

I'm not sure what the `-i` is referring to here... do you have any idea what is causing it?

**Comment by jfy133** [comment]
(note I know this maybe isn't the standard way of using ganon, but I'm using it to add a `ganon` nextflow wrapper for [nf-core/modules](https://github.com/nf-core/modules), with the plan to integrate it into [nf-core/taxprofiler](https://nf-co.re/taxprofiler) - but to get it into modules we need to have a very small CI tests for all the tools to make sure they work)

**Comment by pirovc** [comment]
Thanks for reporting. It's indeed a very silly bug introduce by myself on 1.5.0 and strangely not properly covered by the tests. Will provide a fix still today.

Let me know if you need any help or something is unclear, glad to be included in the nf-core pipeline :)

**Comment by jfy133** [comment]
Hi @pirovc

Thanks for the quick response!

I just hit another issue with 1.4, I will make a separate issue.

**Comment by pirovc** [comment]
Fixed with #246. Let me know if it doesn't work for you. Conda version will be up later today. Further, I'd suggest to use a name prefix, `-d out/` will generate `out/.ibf` and `out/.tax` which I guess are not so desired filenames.

**Comment by jfy133** [comment]
Thank you very much @pirovc ! I'll test once conda recipe is working :), and report back here.

Thanks for the advice too  - don't worry I'm aware. This was just a very minimum test example for the CI test :)
